routes/merged/project/load_balancer.rb

```diff
@@ -6,21 +6,17 @@ class Clover
       load_balancer_list
     end
 
-    r.on api? do
-      r.post String do |lb_name|
-        load_balancer_post(lb_name)
+    r.on web? do
+      r.post do
```

Comment:
```suggestion
      r.post true do
```

Otherwise the route would accept arbitrary garbage in the remaining path.